### ISSUE TITLE
wanna know some details

### ISSUE THREAD

**Issue by powerpun** [AUDIY/FPGA_PCM2DSD#1]
did you test the group delay and THD after conversion?

**Comment by AUDIY** [comment]
No, it has 1st order delta sigma modulator synchronized with I2S BCK.

There's no guarantee.
